yapapi/services/service.py

```diff
@@ -217,11 +218,6 @@ async def start(self) -> AsyncGenerator[Script, Awaitable[List[events.CommandEve
         A clean exit from a handler function triggers the engine to transition the state of the
         instance to the next stage in service's lifecycle - in this case, to `running`.
 
-        On the other hand, any unhandled exception will cause the instance to be either retried on
```

Comment:
well, this paragraph _should_ be still left in here but be reformulated so as not to mention the `respawn_unstarted_instances` argument

Reply:
and should mention `restart_condition` and `reset` instead

Reply:
Brought back and changed